### ISSUE TITLE
PostgreSQL is no longer a platform dependency

### ISSUE THREAD

**Issue by dccboss** [three-Vs/dockerized-hedera-services#1]
From https://github.com/hashgraph/hedera-services/issues/3078

## Steps to Reproduce
1. Pull repository
2. Do not create `.env` file
3. `docker-compose up -d`
4. `docker logs dockerized-hedera-services-node_0-1`

![image](https://user-images.githubusercontent.com/114454594/207559093-519a04c6-b3fb-4eea-b21d-c173d4dff30f.png)

This causes each of the three nodes to each exit immediately with `204 - no error`.

**Comment by 3Nigma** [comment]
Thank you for reporting this! Much appreciated!

Maybe you already doing this, but if this is blocking you in any urgent way, have you thought of using the [official hedera local-node](https://github.com/hashgraph/hedera-local-node) solution? 

We made this repo before anything was officially available and it has helped us greatly, but now that hashgraph has rolled out their own implementation, we switched to using that, mostly.
